narayana-release-process.sh

```diff
@@ -64,22 +64,30 @@ if [[ $M2OK == n* ]]
 then
   exit
 fi
-read -p "You will need: quay.io account with permission to push to jbosstm. Do you have this? y/n: " ENVOK
-if [[ $ENVOK == n* ]]
+read -p "To upload an lra-coordinator image to quay.io you will need a quay.io account with permission to push to jbosstm. Do you have this (if you have access but don't want to upload, answer n* and then the next question)?: " QUAYIOOK
+if [[ $QUAYIOOK == n* ]]
 then
-  exit
+  read -p "Do you have want to continue without uploading lra-coordinator image?" NOQUAYIO
```

Comment:
Thanks Tom for this PR! It allows to release without a quay.io account.
I have only some little typo comments:
Do you have want -> Do you want (line 70)
loging -> logging (line 86)

Reply:
Thank you for the review, I have made updates